nimbusapp

```diff
@@ -140,30 +204,68 @@ if [[ -z ${PROJECT} ]]; then
    PROJECT=`echo ${IMAGE}|sed -r 's|\w+/||'|sed 's/:/_/'`
 fi
 
-#Build Command
-if [[ $ACTION == "inspect" ]]; then
-    DOCKERAPP_ACTION="inspect"
-    ISDOCKERAPPACTION=0
-elif [[ $ACTION == "render" ]]; then
-    ISDOCKERAPPACTION=0
-fi
+COMPOSE_ARGS="$@"
+COMPOSE_DIR="${HOME}/.nimbusapp/cache/${PROJECT}/${REPOSITORY}/${IMAGE}/${VERSION}"
+COMPOSE_FILE="${COMPOSE_DIR}/${IMAGE}.yml"
+COMPOSE_BACKUP="${COMPOSE_FILE}.bk"
 
-COMMAND="docker-app ${DOCKERAPP_ACTION}"
+if [[ -n "${DOCKERAPP_ACTION}" ]]; then
+   if curl -s --connect-timeout "${NIMBUS_DOCKERHUB_TIMEOUT}" "${NIMBUS_DOCKERHUB_URL}" > /dev/null 2>&1; then
+       DOCKERAPP_COMMAND="${DOCKERAPP_COMMAND} ${DOCKERAPP_ACTION} ${DOCKERAPP_ARGS} ${REPOSITORY}/${IMAGE}:${VERSION}"
```

Comment:
This seems to be resulting in an extra space for the render command. Output looks like this:
Running: docker-app render  admpresales/devops:1.1.7.0 > /home/demo/.nimbusapp/cache/devops/admpresales/devops/1.1.7.0/devops.yml

Reply:
I think those are the spaces around the `${DOCKERAPP_ARGS}`, which is empty in the example

Reply:
Meh, let's not worry too much about this one, especially if we're gonna put verbose logging in.